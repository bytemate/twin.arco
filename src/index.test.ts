import { extend } from './index'

describe('Output snapshot', () => {
  it('should got all arco theme variable name', () => {
    expect(Object.keys(extend.colors).sort()).toMatchSnapshot()
  })
})
