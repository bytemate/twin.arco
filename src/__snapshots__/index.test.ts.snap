// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output snapshot should got all arco theme variable name 1`] = `
Array [
  "arco-bg-1",
  "arco-bg-2",
  "arco-bg-3",
  "arco-bg-4",
  "arco-bg-5",
  "arco-bg-popup",
  "arco-bg-white",
  "arco-black",
  "arco-border",
  "arco-border-1",
  "arco-border-2",
  "arco-border-3",
  "arco-border-4",
  "arco-danger-light-1",
  "arco-danger-light-2",
  "arco-danger-light-3",
  "arco-danger-light-4",
  "arco-fill-1",
  "arco-fill-2",
  "arco-fill-3",
  "arco-fill-4",
  "arco-mask-bg",
  "arco-menu-dark-bg",
  "arco-menu-dark-hover",
  "arco-menu-light-bg",
  "arco-neutral-1",
  "arco-neutral-10",
  "arco-neutral-2",
  "arco-neutral-3",
  "arco-neutral-4",
  "arco-neutral-5",
  "arco-neutral-6",
  "arco-neutral-7",
  "arco-neutral-8",
  "arco-neutral-9",
  "arco-primary-light-1",
  "arco-primary-light-2",
  "arco-primary-light-3",
  "arco-primary-light-4",
  "arco-secondary",
  "arco-secondary-active",
  "arco-secondary-disabled",
  "arco-secondary-hover",
  "arco-spin-layer-bg",
  "arco-success-light-1",
  "arco-success-light-2",
  "arco-success-light-3",
  "arco-success-light-4",
  "arco-text-1",
  "arco-text-2",
  "arco-text-3",
  "arco-text-4",
  "arco-tooltip-bg",
  "arco-warning-light-1",
  "arco-warning-light-2",
  "arco-warning-light-3",
  "arco-warning-light-4",
  "arco-white",
  "arcoblue-1",
  "arcoblue-10",
  "arcoblue-2",
  "arcoblue-3",
  "arcoblue-4",
  "arcoblue-5",
  "arcoblue-6",
  "arcoblue-7",
  "arcoblue-8",
  "arcoblue-9",
  "blue-1",
  "blue-10",
  "blue-2",
  "blue-3",
  "blue-4",
  "blue-5",
  "blue-6",
  "blue-7",
  "blue-8",
  "blue-9",
  "cyan-1",
  "cyan-10",
  "cyan-2",
  "cyan-3",
  "cyan-4",
  "cyan-5",
  "cyan-6",
  "cyan-7",
  "cyan-8",
  "cyan-9",
  "danger-1",
  "danger-10",
  "danger-2",
  "danger-3",
  "danger-4",
  "danger-5",
  "danger-6",
  "danger-7",
  "danger-8",
  "danger-9",
  "gold-1",
  "gold-10",
  "gold-2",
  "gold-3",
  "gold-4",
  "gold-5",
  "gold-6",
  "gold-7",
  "gold-8",
  "gold-9",
  "gray-1",
  "gray-10",
  "gray-2",
  "gray-3",
  "gray-4",
  "gray-5",
  "gray-6",
  "gray-7",
  "gray-8",
  "gray-9",
  "green-1",
  "green-10",
  "green-2",
  "green-3",
  "green-4",
  "green-5",
  "green-6",
  "green-7",
  "green-8",
  "green-9",
  "lime-1",
  "lime-10",
  "lime-2",
  "lime-3",
  "lime-4",
  "lime-5",
  "lime-6",
  "lime-7",
  "lime-8",
  "lime-9",
  "magenta-1",
  "magenta-10",
  "magenta-2",
  "magenta-3",
  "magenta-4",
  "magenta-5",
  "magenta-6",
  "magenta-7",
  "magenta-8",
  "magenta-9",
  "orange-1",
  "orange-10",
  "orange-2",
  "orange-3",
  "orange-4",
  "orange-5",
  "orange-6",
  "orange-7",
  "orange-8",
  "orange-9",
  "orangered-1",
  "orangered-10",
  "orangered-2",
  "orangered-3",
  "orangered-4",
  "orangered-5",
  "orangered-6",
  "orangered-7",
  "orangered-8",
  "orangered-9",
  "pinkpurple-1",
  "pinkpurple-10",
  "pinkpurple-2",
  "pinkpurple-3",
  "pinkpurple-4",
  "pinkpurple-5",
  "pinkpurple-6",
  "pinkpurple-7",
  "pinkpurple-8",
  "pinkpurple-9",
  "primary-1",
  "primary-10",
  "primary-2",
  "primary-3",
  "primary-4",
  "primary-5",
  "primary-6",
  "primary-7",
  "primary-8",
  "primary-9",
  "purple-1",
  "purple-10",
  "purple-2",
  "purple-3",
  "purple-4",
  "purple-5",
  "purple-6",
  "purple-7",
  "purple-8",
  "purple-9",
  "red-1",
  "red-10",
  "red-2",
  "red-3",
  "red-4",
  "red-5",
  "red-6",
  "red-7",
  "red-8",
  "red-9",
  "success-1",
  "success-10",
  "success-2",
  "success-3",
  "success-4",
  "success-5",
  "success-6",
  "success-7",
  "success-8",
  "success-9",
  "warning-1",
  "warning-10",
  "warning-2",
  "warning-3",
  "warning-4",
  "warning-5",
  "warning-6",
  "warning-7",
  "warning-8",
  "warning-9",
  "yellow-1",
  "yellow-10",
  "yellow-2",
  "yellow-3",
  "yellow-4",
  "yellow-5",
  "yellow-6",
  "yellow-7",
  "yellow-8",
  "yellow-9",
]
`;
